reference/providers/index.md

```diff
@@ -15,10 +15,10 @@ Provider level.
 
 These are the Cloud Providers currently supported by Spinnaker:
 
-* [App Engine](/reference/providers/)
-* [Amazon Web Services](/reference/providers/)
+* [App Engine](/reference/providers/appengine)
```

Comment:
You should append a trailing `/` character to this link. It saves the user a small 302 redirect.